hledger-lib/Hledger/Reports/AccountTransactionsReport.hs

```diff
@@ -159,28 +168,27 @@ pshowTransactions = pshow . map (\t -> unwords [show $ tdate t, T.unpack $ tdesc
 -- using the provided user-specified report query, a query specifying
 -- which account to use as the focus, a starting balance, a sign-setting
 -- function and a balance-summing function.
```

Comment:
I think "and a balance-summing function" can be dropped, right ?

Reply:
Probably true.